.github/workflows/lint.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
       # Install godot-gdscript-toolkit
       - name: Install godot-gdscript-toolkit
-        run: pip3 install 'gdtoolkit==3.*'
+        run: pip3 install 'gdtoolkit==4.*'
```

Comment:
it seems like you need to also install setuptools since python 3.12, we should pin the python version, so this doesn't happen again. 
```yaml
      - name: Setup Python
        uses: actions/setup-python@v4
        with:
          python-version: 3.12
      - name: Install setuptools
        run: pip3 install setuptools
      # Install godot-gdscript-toolkit
      - name: Install godot-gdscript-toolkit
        run: pip3 install 'gdtoolkit==4.*'
```